web/src/pages/onboarding.tsx

```diff
@@ -221,6 +238,20 @@ const OnboardingPage = (props: Props): ReactElement => {
     }
   };
 
+  const setBusinessPersonaAndRouteToPage = async (flow: FlowType): Promise<void> => {
+    if (flow === undefined) {
+      return;
+    }
```

Comment:
why is this line necessary? According to your type definition in the `types` file, FlowType can never be undefined